### PR TITLE
feat: axelar local dev multiversx

### PR DESCRIPTION
**Pull request by raress96**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,18 @@ Welcome to the Axelar Local Development Environment! This monorepo contains esse
 - **Optional Packages**:
   - [@axelar-network/axelar-local-dev-aptos](./packages/axelar-local-dev-aptos/)
   - [@axelar-network/axelar-local-dev-near](./packages/axelar-local-dev-near/)
+<<<<<<< HEAD
```
Comment (npty) [inline README.md:9]: could you pls clean up merge conflict here?

**Review by npty — COMMENTED** [review]
Thanks for submitting the PR 🙌🏻 
it's lgtm. could you pls also add tests covering the relay between evm and multiversx? You can looks at `aptos.spec.ts` or `near.spec.ts` as a reference.

**Comment by raress96** [comment]
> Thanks for submitting the PR 🙌🏻 it's lgtm. could you pls also add tests covering the relay between evm and multiversx? You can looks at `aptos.spec.ts` or `near.spec.ts` as a reference.

Hello, I have added tests for MultiversX. Also updated the pipeline but not 100% sure if it will run correctly, if you can trigger it that would be great and I can update it accordingly.

**Comment by npty** [comment]
> > Thanks for submitting the PR 🙌🏻 it's lgtm. could you pls also add tests covering the relay between evm and multiversx? You can looks at `aptos.spec.ts` or `near.spec.ts` as a reference.
> 
> Hello, I have added tests for MultiversX. Also updated the Github Action but not 100% sure if it will run correctly, if you can trigger it that would be great and I can update it accordingly.

@raress96 
Sorry, I've missed this. I've tried Github Action, it failed at setup-python step. Could you try updating the working directory?

**Comment by raress96** [comment]
> > > Thanks for submitting the PR 🙌🏻 it's lgtm. could you pls also add tests covering the relay between evm and multiversx? You can looks at `aptos.spec.ts` or `near.spec.ts` as a reference.
> > 
> > 
> > Hello, I have added tests for MultiversX. Also updated the Github Action but not 100% sure if it will run correctly, if you can trigger it that would be great and I can update it accordingly.
> 
> @raress96 Sorry, I've missed this. I've tried Github Action, it failed at the `setup-python` step. Could you try updating the working directory?

The pipeline should be fixed.

**Review by npty — APPROVED** [review]

**Pull request merged**